blivet/osinstall.py

```diff
@@ -1341,10 +1360,89 @@ def _update_custom_storage_ksdata(self):
         self.fsset = FSSet(self.devicetree)
 
     def reset(self, cleanup_only=False):
+        """ Reset storage configuration to reflect actual system state.
+
+            This will cancel any queued actions and rescan from scratch but not
+            clobber user-obtained information like passphrases, iscsi config, &c
+
+            :keyword cleanup_only: prepare the tree only to deactivate devices
+            :type cleanup_only: bool
+
+            See :meth:`devicetree.Devicetree.populate` for more information
+            about the cleanup_only keyword argument.
+        """
+        if flags.installer_mode:
```

Comment:
Just out of curiosity, is the plan for the future to assume this to be ``True`` if/when the ``InstallerStorage`` is being used?